xonsh/prompt_toolkit_key_bindings.py

```diff
@@ -32,3 +32,11 @@ def _(event):
         indent instead of autocompleting.
         """
         event.cli.current_buffer.insert_text(env.get('INDENT'))
+
+    @handle(Keys.BackTab)
+    def _(event):
```

Comment:
Thanks @gforsyth - can you please give this function a more meaningful name, even if it is private? Thanks!